assets/stylesheets/shared/mixins/_breakpoints.scss

```diff
@@ -19,13 +19,15 @@ $breakpoints: 480px, 660px, 800px, 960px, 1040px, 1280px, 1400px; // Think very
 					@media ( max-width: $breakpoint ) {
 						@content;
 					}
-				} @else {
+				}
+ @else {
 					@if $size == $and-larger {
 						$approved-value: 2;
 						@media ( min-width: $breakpoint + 1 ) {
 							@content;
 						}
-					} @else {
+					}
+ @else {
```

Comment:
These breaks before an else aren't aligned with our typical style. Can that be improved so the else is on the same line as braces?

Reply:
Nice spotting @sirreal :wink: Turns out this can be indeed improved by configuring the [block-closing-brace-newline-after](https://stylelint.io/user-guide/rules/block-closing-brace-newline-after/) rule. The `@if` and `@else` SCSS statements are actually an example in the docs.

I updated the Stylelint config in 4a4a8be and updated the file formatting in 3fa79b6.